content/mooreslaw-tutorial.md

```diff
@@ -455,7 +414,7 @@ np.savez(
     transistor_count=transistor_count,
     transistor_count_predicted=transistor_count_predicted,
     transistor_Moores_law=transistor_Moores_law,
-    regression_csts=AB,
+    regression_csts=(A, B),
```

Comment:
unrelated again, and an extreme nitpick, but the notes strings in the cell above look a bit stupid. The cells are long and having the sidebar, yet `B, A` and the `)` are rendered in a practically empty line.

Reply:
Thanks for the feedback - I think there are definitely other changes we could consider here. Attaching metadata to data files to describe the underlying data is a good practice, but I'm not sure that it's something you'd necessarily want to advertise doing with npz files - there are much more full-featured file formats with built-in support for metadata (e.g. HDF5).

Either way, I intend (and encourage others) to make such suggestions, but will do so in separate PRs so this one can stay focused on the `statsmodels` -> `np.polynomial` transition!